### DISCUSSION TITLE
How can I use .vue file on extension?

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [ueberdosis/tiptap#866] [General Questions] [answered]
Currently I can render vue on my custom extension using `view()`. I'm curious to see if I can use vue file instead of directly writing it on `view()`

**Comment by hanspagel** [comment, marked as answer]
That’s currently not supported, sorry.

**Comment by daffamumtaz2361** [comment]
Ah, that's too bad. But could you answer my other question, [why can't I click link rendered by vue?](https://github.com/ueberdosis/tiptap/discussions/867)
